src/test/java/de/tum/in/www1/artemis/tutorialgroups/AbstractTutorialGroupIntegrationTest.java

```diff
@@ -333,7 +332,7 @@ public static class RandomTutorialGroupGenerator {
 
         private static final String ALL_CHARS = LOWERCASE_LETTERS + NUMBERS;
 
-        private static final SecureRandom RANDOM = new SecureRandom();
+        private static final Random RANDOM = new Random(1042001L);
 
         public static String generateRandomTitle() {
```

Comment:
I had a look into removing the RandomTutorialGroupGenerator and it would be a lot more complicated than removing the RandomConversationNameGenerator was.
This is why instead, I have chosen to replace SecureRandom with Random and to give it a seed. This way, we should get the same "random" numbers with each test run, yet it was an easy fix :)